tests/typescript_keywords.ts

```diff
@@ -0,0 +1,5 @@
+export { Abc } from './foobar/abc';
```

Comment:
these all sit directly in `tests` which should be avoided.. what are they for anyway? these don't seem to come from the schemas we use with other languages

Reply:
seems it originates from https://github.com/google/flatbuffers/pull/7161 with the purpose of testing typescript keyword clashes or something like that

Reply:
I agree it's messy that generated stuff ends up directly in tests. However, note that is also true for other langs fx. https://github.com/google/flatbuffers/blob/master/tests/monster_test_generated.h.

Reply:
That is from a language which generates all its code (including namespaces) into a single file. Languages that create a file per class and directories for namespaces should use directories in general.

Reply:
but the main reason for generated files not ending up in a subdir is that they do not have namespace?

Reply:
I'll note that the `_generated.ts` files are generating all their code in a single file (i.e., all the ones generated with `--ts-flat-files`).

For the "normal" codegen'd files (e.g., the `typescript_keywords.ts`), I'm not as familiar with the history of why this setup was chosen, but the non-folder'd file is serving as a sort of equivalent to an `index.ts` where it doesn't contain any actual definitions, and just exports all the relevant types for a given schema file, and so it makes some logical sense to me that it would not be in a random sub-folder.